serverless/src/routes/webhooks/slack.ts

```diff
@@ -126,16 +126,27 @@ slackWebhookRouter.post('/recieved_event', async (req: Request, res: Response, n
       if(userIds) {
         console.log('mentioned!');
         console.log(event);
+        const srcUserDisplayName = await getUserDisplayName(event.user);
+        console.log('srcUserDisplayName');
+        console.log(srcUserDisplayName);
 
-        const newRecords = [];
-        userIds.map(userId => {
-          newRecords[newRecords.length] = {
+        await Promise.all(userIds.map(async userId => {
```

Comment:
ここはPromise記法を使わずに、async/awaitだけで書けないのでしょうか？
```suggestion
        userIds.map(async userId => {
```

あんまりJS詳しくないので、間違っていたら「それはそういうもんなんです」でOKです。勉強しておきます。

Reply:
私もよくわかっていなかったのですが、配列のmap処理の中でasync関数を呼ぶ場合は、値としてPromiseが返らないようにPromise.allを使う必要があるようです。
https://naruhodoweb.com/javascript-map-async/

Reply:
なるほど！勉強になりました。ありがとうございます！